### ISSUE TITLE
node classification 

### ISSUE THREAD

**Issue by jcyl** [Sherrylone/M-Mix#2]
Hi, I have some question in node classification task. 
(1)In line 53 of node/model.py, why shouldn't be like this:
                out = self.to_v(torch.bmm(attention, out))
(2)and the object of trainning is set to be CrossEntropyLoss between 'new_label' and 'logits', which does not match the description in Section 3.3. Moreover, it is different from the implement of 'mvgrl'. I hope to know your consideration in this regard.

**Comment by Sherrylone** [comment]
Hi, I uploaded the latest code version, so there may be some differences (we attempt many trials after the submission). 
(1). Yes, it should be that.
(2). We try CE loss and find CE loss gets similar results to BCELoss (in vision, most contrastive-based methods use CE loss). You can simply switch to BCE loss and modify the format of the label to get the accuracy reported in the paper.

**Comment by jcyl** [comment]
Thanks for your explaination. In your paper, the only hyper-parameter tuned for node classification is batchsize. It seems that you calculate the accuracy every 10 epoch and report the best. Do I have an exact understanding?

**Comment by Sherrylone** [comment]
We also tune dimensions (see paper). Besides, the frequent accuracy log is used to plot the curve in Fig.3. The reported accuracy in Table follows the evaluation protocol in MVGRL.

**Comment by jcyl** [comment]
thx a lot
